app.rb

```diff
@@ -0,0 +1,108 @@
+require_relative 'leshoyasoap'
+require 'colorize'
+# Ask for a username
+
+
+a =  Dir.glob("/Users/leshoyastarkey/Dropbox/SoapBox/*")
+
+a. each do |element|
+
+  file  = File.open(element)
+
+
+  answers = {username: file.read,
+  body: File.basename(element),
+  created_at: file.birthtime}
+
+  puts answers[:username]
+  puts answers[:body]
+  puts answers[:created_at]
+  puts "-" * 50
+end
+
+#Asks for My Username + saves the information in a local file
```

Comment:
Nice use of comments in this file. Makes it very readable (and good for future reference)